content/posts/tau/index.md

```diff
@@ -18,12 +18,92 @@ summary: "A customize osu! ruleset with a circle surrounding a paddle and some n
 - GitHub Repositories : [Altenhh/tau](https://github.com/Altenhh/tau/)
 - Discord : [tau](https://discord.com/invite/7Y8GXAa)
 
-Tau is a custom ruleset with a circle surrounding a paddle and some notes. Based on idea from [pizzazip's Tau](https://deadlysprinklez.itch.io/tau). The ruleset is developed by Laica.
+Tau is a custom ruleset with a circle surrounding a paddle and some notes.
+
+ *The idea is based on [pizzazip's Tau](https://deadlysprinklez.itch.io/tau). The ruleset is developed by [Laica](https://github.com/Altenhh).*
 
 # gameplay
 
 Tau's gameplay revolves around a paddle which is moved around a circle with the mouse. In the circle, there are different types of notes trying to escape it. The player has to move the paddle to block these notes, then hit them using keyboard keys.
-You can see all note types in [Tau gameplay note types wiki](https://github.com/Altenhh/tau/wiki/Note-types).
+
+The gameplay playfield looks like the following:
+
+![Playfield](Images/Playfield.png)
+
+This is an empty playfield, the parts it contains are the following:
+
+![Cursor](Images/Cursor.png) ![Paddle](Images/Paddle.png)
+
+ This is the cursor   /       This is the paddle
+  1.  The Cursor
+    This is your mouse, it shows you where on the screen your mouse currently is.
+    
+  *The closer the cursor is to the middle of the screen the faster your paddle moves, the further away the cursor is from the middle of the playfield the slower your paddle moves (Because the circle you move your mouse in becomes smaller) keep this in mind while playing to make it easier to play*
+  2. The Paddle 
+    You move it by moving your mouse and cursor, you use it to hit the notes and not let them escape your grasp.
+    *You should always move it to the place the next note you are gonna hit is heading and keep it there until the note intersects the paddle and you press the button required for that specific note.*
+
+
+
+# note types
+
+![Gameplaymods](Images/HeaderNotes.png)
+
+> Source: [Tau note types wiki](https://github.com/Altenhh/tau/wiki/Note-types).
+
+## Beats
+
+![Beats](Images/Beats.png)
+
+These notes will appear from the center and move towards the ring. Move the paddle to the place they are going towards and hit them as they intersect the ring to maximize score.
+
+*You can hit "Beats" using the mouse left and right buttons or using the Z and X buttons in your keyboard*
+
+*And you can easily customize that keybind to another key in the settings of osu!*
+
+
+| Judgement | Score Value | Accuracy |
+| --------- | ----------- | -------- |
+| Miss      | 0           | 0%       |
+| Good      | 200         | 33.66%   |
+| Great     | 300         | 100%     |
```

Comment:
```suggestion
{{< table "table table-striped" >}}
| Judgement | Score Value | Accuracy |
| --------- | ----------- | -------- |
| Miss      | 0           | 0%       |
| Good      | 200         | 33.66%   |
| Great     | 300         | 100%     |
{{</ table >}}
```
Because this website use bootstrap so we use this tag (Open `{{< table "table table-striped" >}}` and close with `{{</ table >}}` to make Hugo (this website's framework) working with a table css styling.